clocks/users/redis_client.py

```diff
@@ -0,0 +1,22 @@
+from django.core.cache import cache
+
+
+def save_new_client_to_cache(token, nickname: str, role: str, ttl: int = 432000) -> None:
+    """
+    Сохранение данных клиента в кэш с использованием token.
+    """
+    cache.set(token, {"nickname": nickname, "role": role}, ttl)
+
+
+def check_token_in_cache(token: str) -> bool:
+    """
+    Проверка на существование записи в кэше.
+    """
+    return cache.get(token) is not None
```

Comment:
можно проще без взятия объекта
```suggestion
    return bool(cache.ttl(token))
```